ocaml-lsp-server/src/compl.ml

```diff
@@ -380,6 +443,7 @@ let resolve doc (compl : CompletionItem.t) (resolve : Resolve.t) query_doc
                 (Document.Merlin.source doc)
                 logical_position
             in
+            logCompletion @@ "completion prefix is:" ^ prefix;
```

Comment:
This log message is allocating its argument if no logging is enabled. Defeating the purpose of having `Log.log` taking a function argument.

Reply:
Yeah, good point